packages/dx-react-scheduler/docs/reference/drag-drop-provider.md

```diff
@@ -35,6 +35,7 @@ Name | Type | Default | Description
 -----|------|---------|------------
 allowDrag? | (appointmentData: [AppointmentModel](./scheduler.md#appointmentmodel)) => boolean | () => true | A function that specifies draggable appointments.
 allowResize? | (appointmentData: [AppointmentModel](./scheduler.md#appointmentmodel)) => boolean | () => true | A function that specifies resizable appointments.
+scrollSpeed? | number | 15 | Specifies the scroll speed when resizing or dragging an appointment.
```

Comment:
> Specifies the scroll speed when resizing or dragging an appointment.

Specifies the auto-scroll speed while appointment drag-and-drop manipulations.

Reply:
```suggestion
scrollSpeed? | number | 15 | Specifies the scroll speed during appointment resize or drag and drop.
```